imap/jmap_calendar.c

```diff
@@ -2578,6 +2578,12 @@ static int setcalendarevents_create(jmap_req_t *req,
     const char *calendarId = NULL;
     strarray_t schedule_addresses = STRARRAY_INITIALIZER;
 
+    static int icalendar_max_size = -1;
```

Comment:
Nit: I might have missed during the review for CardDAV: it's a pity that libconfig only supports signed int, which on my machine is a 4-byte signed integer. I guess any meaningful event will be way smaller that 2GB. Should we enforce an explicit maximum?

Reply:
Not sure what you mean by explicit maximum.  Do you mean within libconfig?

Reply:
I meant to state a maximum in the doc. But it's really not that important.